core/invoker/src/main/scala/whisk/core/invoker/Invoker.scala

```diff
@@ -221,11 +223,11 @@ class Invoker(
         implicit transid: TransactionId): Unit = {
         error(this, errorMsg)
         val msg = tran.msg
-        val name = EntityName(actionDocInfo.id().split(EntityPath.PATHSEP)(1))
+        val name = EntityPath(actionDocInfo.id.id)
```

Comment:
I thought we agreed to leave this as is. Instead, fix the namespace value when creating the activation to not be `subject`.